app/src/main/java/ingsw/group1/findmyphone/alarm/AlarmManager.java

```diff
@@ -0,0 +1,61 @@
+package ingsw.group1.findmyphone.alarm;
+
+import android.content.Context;
+import android.media.AudioManager;
+import android.media.MediaPlayer;
+import android.provider.Settings;
+import android.util.Log;
+
+/**
+ * Manager that starts an alarm with the default ringtone of this device
+ * For the composition of request and response message
+ * it uses methods of {@link AlarmMessageParser}
+ *
+ * @author
+ * @author Giorgia Bortoletti (refactoring)
+ */
+public class AlarmManager {
+
+    public final String ALARM_MANAGER_TAG = "AlarmManagerTag";
+
+    MediaPlayer mediaPlayer;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/RicDeZen/ingsw-group1-findmyphone/pullRequest?prid=4929934)